### PR TITLE
Sejer.BiblioManuels version 1.0.0.0

### PR DESCRIPTION
**Pull request by tbringuier**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/14377)

### CODE REVIEW

File: manifests/s/Sejer/BiblioManuels/1.0.0.0/Sejer.BiblioManuels.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Sejer.BiblioManuels
+PackageVersion: 1.0.0.0
```
Comment (ItzLevvie) [inline manifests/s/Sejer/BiblioManuels/1.0.0.0/Sejer.BiblioManuels.installer.yaml:4]:
```suggestion
PackageVersion: 3.5.0
```

File: manifests/s/Sejer/BiblioManuels/1.0.0.0/Sejer.BiblioManuels.locale.fr-FR.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Sejer.BiblioManuels
+PackageVersion: 1.0.0.0
```
Comment (ItzLevvie) [inline manifests/s/Sejer/BiblioManuels/1.0.0.0/Sejer.BiblioManuels.locale.fr-FR.yaml:4]:
```suggestion
PackageVersion: 3.5.0
```

File: manifests/s/Sejer/BiblioManuels/1.0.0.0/Sejer.BiblioManuels.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Sejer.BiblioManuels
+PackageVersion: 1.0.0.0
```
Comment (ItzLevvie) [inline manifests/s/Sejer/BiblioManuels/1.0.0.0/Sejer.BiblioManuels.yaml:4]:
```suggestion
PackageVersion: 3.5.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/Sejer/BiblioManuels/1.0.0.0/Sejer.BiblioManuels.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/s/Sejer/BiblioManuels/1.0.0.0/Sejer.BiblioManuels.installer.yaml:4]: You'll also need to rename the folder `1.0.0.0` to `3.5.0` to make sure that it matches the version shown in the Control Panel.

File: manifests/s/Sejer/BiblioManuels/1.0.0.0/Sejer.BiblioManuels.locale.fr-FR.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Sejer.BiblioManuels
+PackageVersion: 1.0.0.0
+PackageLocale: fr-FR
+Publisher: Sejer
+PackageName: Biblio Manuels
+License: Copyright Sejer
```
Comment (ItzLevvie) [inline manifests/s/Sejer/BiblioManuels/1.0.0.0/Sejer.BiblioManuels.locale.fr-FR.yaml:8]:
```suggestion
License: Copyright (c) 2021 Sejer
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by MichelBaie** [comment]
ok 👍

**Comment by denelon** [comment]
@MichelBaie when the version number is changed in the manifest, the directory structure also needs to reflect the same version. I think your PR is getting very close 👍

**Pull request closed without merging**